gf-ui-lib/src/components/Map/mapLeaflet/MapLeaflet.tsx

```diff
@@ -0,0 +1,50 @@
+import {useContext, useEffect} from "react";
+import {MapContainer, TileLayer, ZoomControl, useMap} from "react-leaflet";
+import {MarkersLeaflet} from "./markersLeaflet/MarkersLeaflet";
+import {MapContext} from "../MapContext";
+import "leaflet/dist/leaflet.css";
+import styles from "./MapLeaflet.module.scss";
+
+const MAP_CENTER_X = 42.24625;
+const MAP_CENTER_Y = 42.68017;
+const MAP_DEFAULT_ZOOM = 15;
+
+interface ICenterOnMarkerProps {
```

Comment:
If i'm not mistaken in our project we use namin for interfaces without I letter. For example, here will be interface CenterOnMarkerProps. Please check everywhere

Reply:
Done for map. But we have a few interface names started from the I letter in gf-webapp-admin